superset-frontend/webpack.config.js

```diff
@@ -20,17 +20,17 @@
 const fs = require('fs');
 const path = require('path');
 const webpack = require('webpack');
-const BundleAnalyzerPlugin = require('webpack-bundle-analyzer')
-  .BundleAnalyzerPlugin;
+const { BundleAnalyzerPlugin } = require('webpack-bundle-analyzer');
 const { CleanWebpackPlugin } = require('clean-webpack-plugin');
 const CopyPlugin = require('copy-webpack-plugin');
 const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 const OptimizeCSSAssetsPlugin = require('optimize-css-assets-webpack-plugin');
 const SpeedMeasurePlugin = require('speed-measure-webpack-plugin');
 const TerserPlugin = require('terser-webpack-plugin');
-const WebpackAssetsManifest = require('webpack-assets-manifest');
+const ManifestPlugin = require('webpack-manifest-plugin');
```

Comment:
Replaced [WebpackAssetsManafestPlugin](https://github.com/webdeveric/webpack-assets-manifest) with [WebpackManifestPlugin](https://github.com/danethurber/webpack-manifest-plugin) because the latter is recommended by Webpack's official docs and seem to have more users.

Reply:
sgtm